apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.js

```diff
@@ -120,20 +127,25 @@ const CreatePaymentByLinkService = new GQLCustomSchema('CreatePaymentByLinkServi
                 }
 
                 const payForQR = async (lastBillingReceipt) => {
+                    if (!lastBillingReceipt) {
```

Comment:
please don't create a new function for every request. Could you please extract it as a normal function?

Reply:
I need to pass this function as a resolver to another asynchronous function. I also need to have rw-access to context and other variables outside. The closure is good for that.